src/content/1.7/scale/_index.md

```diff
@@ -38,3 +38,12 @@ It may not be as simple as it sounds, as PrestaShop requires many files being sh
 Also, once you have set up your first server, setting additional ones should be far easier.
 
 The good thing is that, though requiring more sweat at first, horizontal scaling is better in terms of performances. Done well, adding a second server may easily double your application performance, if not more.
+
+### Further than LAMP
+
+Another way to scale your shop is to introduce other systems to improve your performances. Here is a quicklist of what can be used:
+
+1. Using a reverse proxy such as varnish for Front Office
+2. Using Redis to store sessions
+3. Use elasticsearch or a solution built on it for processing customer searches
+4. Use a CDN for static files
```

Comment:
On a development environment an http cache can be tedious especially for front-end developers. I wonder if it is useful to specify that it is recommended in a production environment only?

Reply:
It's always good to be more accurate 👍

Reply:
Fixed in https://github.com/PrestaShop/docs/pull/792/commits/17de842062f4228fc27a9eebfc3513b43c0b04e4